### DISCUSSION TITLE
Run Next State ConversationHandler

### DISCUSSION THREAD

**Discussion by lucasmolinari** [python-telegram-bot/python-telegram-bot#4030] [Q&A]
Is there a way to go to the next Conversation State without receiving an user update?

**Comment by Bibo-Joshi** [comment]
No

**Comment by aelkheir** [comment]
The way I solve this is by manually calling a next state's `next_callback` function that I want to run while still in a `current_callback`. 

You should also remember to return - from the current callback - whatever the next callback returns. That is incase you want to keep the flow of the  conversation as normal.

```python
# inside a current_callback
return next_callback(update, context)
```
Of course this should be wrapped in a conditional of some kind.

There's just one caveat. When running next callback like this, `update` and `context` will understandably be the same objects in between the functions. So inside `next_callback` you can't trust properties like `update.text, callback_query.data, context.match`, ..etc to be what you would normally expect.
